hstream/src/HStream/Server/Core/Cluster.hs

```diff
@@ -83,11 +83,8 @@ describeCluster ServerContext{gossipContext = gc@GossipContext{..}, ..} = do
 
     updateServerVersion version node = node { serverNodeVersion = Just version }
 
-lookupResource :: ServerContext -> LookupResourceRequest -> IO ServerNode
-lookupResource sc LookupResourceRequest{..} = do
-  case lookupResourceRequestResType of
-    Enumerated (Right rType) -> lookupResource' sc rType lookupResourceRequestResId
-    x -> throwIO $ HE.InvalidResourceType (show x)
+lookupResource :: ServerContext -> ResourceType -> T.Text -> IO ServerNode
+lookupResource = lookupResource'
```

Comment:
Just remove this function, and rename the origin `lookupResource'` to `lookupResource`